modules/Makefile.docker-machine

```diff
@@ -0,0 +1,63 @@
+DOCKER_MACHINE_NAME ?= gladly
+DOCKER_DOWNLOAD_URL ?= https://get.docker.com/builds/$(LOCAL_OS)/$(LOCAL_ARCH)/docker-$(DOCKER_VERSION)
+DOCKER_ISO ?= https://github.com/boot2docker/boot2docker/releases/download/v$(DOCKER_VERSION)/boot2docker.iso
+DOCKER_MACHINE_MEMORY ?= 4096
+DOCKER_MACHINE_DISK ?= 50000
+
+export DOCKER_MACHINE_NAME
+
+DEPS_TARGETS += machine:deps
+
+.PHONY : machine\:deps machine\:info machine\:recreate machine\:create machine\:destroy machine\:start machine\:stop machine\:restart machine\:deps
+
+## Setup dependencies for docker-machine
+machine\:deps:
+	@[ -x $(DOCKER_CMD) ] || (curl -s -o $(DOCKER_CMD) $(DOCKER_DOWNLOAD_URL) && chmod 755 $(DOCKER_CMD))
+	@(docker-machine status $(DOCKER_MACHINE_NAME) 2>&1 | grep -q "does not exist"; [ $$? -ne 0 ]) || $(SELF) machine:create
+	@(docker-machine status $(DOCKER_MACHINE_NAME) 2>&1 | grep -q "^Running") || $(SELF) machine:start
+	@docker-machine env $(DOCKER_MACHINE_NAME) | tr -d '"' > "$(MAKEFILE_PATH).env"
+	$(eval -include $(MAKEFILE_PATH).env)
+	@$(call assert,DOCKER_HOST)
```

Comment:
Might have an ordering problem here. If you delete Makefile.env then run `make deps` it will fail on this line subsequent run is ok

Adding a bit of debug, it appears the include is happening before the docker-machine commands